### ISSUE TITLE
Turning off console logging

### ISSUE THREAD

**Issue by Gnindit** [austinsc/node-red-contrib-envisalink-dsc#2]
Hi
Just installed Envisa link to my DSC and using it with node-red.

Is it possible to somehow turn off all the logging to the console without changing teh node.red settings to log less?

**Comment by Gnindit** [comment]
Added this to  envisalink-controller.js

    this.el.on('log-trace', function(text) {
      _this.trace(RED._(text));
    });

Then I could change log-debug to log-trace in envisalink.js where I wanted.

**Comment by austinsc** [comment]
Good idea- I'll integrate that or if you create a pull request, I'll merge it.
